Components/BlockPostHeader/_style.scss

```diff
@@ -0,0 +1,109 @@
+[is='flynt-block-post-header'] {
+
+  // ##BLOCK-POST-HEADER
+  // =============================================================
+
+  // SPACING
+  // -------------------------------------------------------------
+
+  .container {
+    @include center;
+  }
+
+  .blockPostHeader {
+    padding-top: 1.125rem;
+
+    @media (min-width: $breakpoint-mobile-horizontal) {
+      padding-top: 1.5rem;
+    }
+
+    @media (min-width: $breakpoint-tablet) {
+      padding-top: 2rem;
+    }
+
+    &-meta,
+    &-media,
+    &-intro {
+      margin: 1.375rem 0 0;
+
+      @media (min-width: $breakpoint-tablet) {
+        margin-top: 1.625rem;
+      }
+
+      &:first-child {
+        margin-top: 0;
+      }
+    }
+
+    &-title,
+    &-meta,
+    &-intro {
+      margin-left: auto;
+      margin-right: auto;
+      max-width: $content-max-width;
+    }
+
+    &-media {
+      .image {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 800px;
+      }
+    }
+
+    &-meta + .blockPostHeader-intro {
+      margin-top: 1rem;
+
+      @media (min-width: $breakpoint-tablet) {
+        margin-top: 1.125rem;
+      }
+    }
+  }
+
+  // STYLE
+  // -------------------------------------------------------------
+
+  .blockPostHeader {
+    &-h1 {
+      font-size: 2rem;
+      line-height: 1.175;
+      margin: 0;
+      padding: 0;
+
+      @media (min-width: $breakpoint-mobile-horizontal) {
+        font-size: 2.5rem;
+      }
+
+      @media (min-width: $breakpoint-tablet) {
+        font-size: 2.75rem;
+      }
+    }
+
+    &-meta {
+      color: $color-text;
+      font-size: 14px;
+      line-height: 1.5;
+    }
+
+    &-category {
+      margin-right: 0.25rem;
+    }
+
+    &-readingtime {
+      margin-left: 0.25rem;
+    }
+
+    &-intro {
+      p {
+        font-size: 1.125rem;
```

Comment:
I would prefer using one of our headline classes instead, like at the top in the BaseStyle.